src/olympia/reviewers/tests/test_utils.py

```diff
@@ -1492,8 +1492,18 @@ def _test_block_multiple_unlisted_versions(self, redirect_url):
         assert logs[0].created == logs[1].created
 
         # We should have set redirect_url to point to the Block admin page
-        assert self.helper.redirect_url == redirect_url % (
-            old_version.version, self.version.version)
+        if '%s' in redirect_url:
+            redirect_url = redirect_url % (
+                old_version.version, self.version.version)
+        assert self.helper.redirect_url == redirect_url
+
+    def test_pending_blocksubmission_multiple_unlisted_versions(self):
+        subm = BlockSubmission.objects.create(
+            input_guids=self.addon.guid, updated_by=user_factory())
+        redirect_url = (
+            reverse('admin:blocklist_blocksubmission_change', args=(subm.id,)))
+        assert Block.objects.count() == 0
```

Comment:
I don't understand why this assert is here.

Reply:
it's checking that just visiting the url isn't creating a Block - you would need to take a further action to create something.